react-basic-practice/src/pages/dashboard.tsx

```diff
@@ -35,20 +35,21 @@ const Dashboard: React.FC = () => {
   const handleAction = (type: ActionType, id?: number) => {
     const customer = id && data.find((item) => item.id === id);
 
-    customer &&
+    if (customer) {
       handleOpenModal(
         type === "Edit"
           ? MODAL_TITLES.EDIT_CUSTOMER
           : MODAL_TITLES.VIEW_CUSTOMER,
         customer
       );
+    }
   };
 
   const renderModalContent = () =>
     modalTitle === MODAL_TITLES.VIEW_CUSTOMER && viewData ? (
       <CustomerView data={viewData} />
     ) : (
-      <CustomerForm />
+      <CustomerForm data={viewData} />
```

Comment:
`viewData` is not a good name. Pls change to `customer` or something like that. It's just true for case viewData

Reply:
i done